src/scss/core/_grid.scss

```diff
@@ -6,7 +6,7 @@
 }
 
 @include media-breakpoint-up(lg) {
-	.container {
+	.container, .container-fluid {
```

Comment:
why do we need padding on container-fluid? in this way it's not gonna be full-width

Reply:
actually, `.container-fluid` also have gutter padding. What I did here is to adjust it to have desktop gutter with in lg screen.